README.md

```diff
@@ -192,39 +240,6 @@ predict_linear(aws_es_free_storage_space_minimum[2d], 86400 * 7) + on (name) gro
 ((increase(yace_cloudwatch_requests_total[10m]) * 6 * 24 * 32) - 100000) / 1000 * 0.01
 ```
 
-## Required IAM permissions
```

Comment:
I thought it made sense to keep this breakdown with the example policy vs a separate section further down